script.test/main.py

```diff
@@ -108,7 +108,9 @@ def start(addon_path, repo_addons, config=None):
             _check_for_legacy_strings_xml(addon_report, addon_path)
```

Comment:
We should probably move away from the idea, that the test script needs to be in sync or even is the same python source. I would prefer it to be just some random addon.

Reply:
Yes, I also don't like that kind of code duplication. We probably want to test this with a bunch of addons having known problems.